README.md

```
@@ -19,154 +19,73 @@ _A Sleek & Lightweight VLC Alternative with On-Device AI Subtitles_
 
 ## ✨ Features
 
-### 🎬 Cinematic Mode
-Enjoy your media with a beautifully blurred background and centered content for truly immersive viewing.
+### 🤖 On-Device AI & Subtitles
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)